config/config.go

```diff
@@ -12,9 +12,12 @@ const KafkaTLSProtocol = "TLS"
 // Config represents service configuration for dp-search-data-extractor
 type Config struct {
 	BindAddr                   string        `envconfig:"BIND_ADDR"`
+	EnableTopicCache           bool          `envconfig:"ENABLE_TOPICS_CACHE"`
+	TopicCacheUpdateInterval   time.Duration `envconfig:"TOPICS_CACHE_UPDATE_INTERVAL"`
```

Comment:
```suggestion
	TopicCacheUpdateInterval   time.Duration `envconfig:"TOPIC_CACHE_UPDATE_INTERVAL"`
```
Let's be consistent on no plurals. 'Topic API', 'Topic Cache' etc.

Reply:
missed this. Apologies